client/src/utils/api.js

```diff
@@ -13,3 +13,19 @@ export async function getProjectById(id) {
   const project = await response.json();
   return project;
 }
+
+export async function getProjects() {
+  const response = await fetch("/api/projects?_sort=id&_order=desc");
+  const projects = await response.json();
+  return projects;
+}
+export async function getLimitedProjects() {
+  const response = await fetch("/api/projects?_sort=id&_order=desc&_limit=20");
+  const projects = await response.json();
+  return projects;
+}
+export async function getCategory(category) {
+  const response = await fetch(`/api/projects?data.category=${category}`);
+  const projects = await response.json();
+  return projects;
```

Comment:
mabye making one function

query = "?_sort=id&_order=desc"
query = "?_sort=id&_order=desc&_limit=20"
query =  "?data.category=${category}"

```suggestion
export async function getData (query) {
  const response = await fetch(`/api/projects${query}`);
  const data = await response.json();
  return data;
}
```